errors/runconfig.go

```diff
@@ -0,0 +1,204 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/runconfig component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeInvalidMacAddress is generated when invalid Mac Address is provided.
+	ErrorCodeInvalidMacAddress = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDMACADDRESS",
+		Message:        "%s is not a valid mac address",
+		Description:    "The specified Mac Address is invalid",
+		HTTPStatusCode: http.StatusNotFound,
+	})
+
+	// ErrorCodeInvalidMemorySwappiness is generated when invalid Memory Swappiness value is provided.
+	ErrorCodeInvalidMemorySwappiness = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDMEMORYSWAPPINESS",
+		Message:        "Invalid value: %d. Valid memory swappiness range is 0-100",
+		Description:    "The specified Memory swappiness is invalid",
+		HTTPStatusCode: http.StatusNotFound,
```

Comment:
Should be status 422, which unfortunately doesn't have a var defined in the http package.